### PR TITLE
feat: remove old tables and their logic

### PR DESCRIPTION
**Pull request by capJavert**
Removing tables and old mutations that are no longer used. 

[AS-27](https://dailydotdev.atlassian.net/jira/software/projects/AS/boards/30?selectedIssue=AS-27)

[AS-27]: https://dailydotdev.atlassian.net/browse/AS-27?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/migration/1704816100142-RemoveUpvoteDownvoteHiddenPostTables.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class RemoveUpvoteDownvoteHiddenPostTables1704816100142 implements MigrationInterface {
+    name = 'RemoveUpvoteDownvoteHiddenPostTables1704816100142'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`DROP TABLE "upvote"`);
+      await queryRunner.query(`DROP TABLE "downvote"`);
+      await queryRunner.query(`DROP TABLE "hidden_post"`);
```
Comment (omBratteng) [inline src/migration/1704816100142-RemoveUpvoteDownvoteHiddenPostTables.ts:9]: This is always scary 😬

Comment (omBratteng) [reply src/migration/1704816100142-RemoveUpvoteDownvoteHiddenPostTables.ts:9]: Like imagine, what if there's a really edge case utf-8 bug where `DROP TABLE "upvote"` for some reason gets parsed as `DROP DATABASE` 🤯 
Worst nightmare.

**Review by omBratteng — APPROVED** [review]
Haha, lines of code goes brrrt

File: src/migration/1704816100142-RemoveUpvoteDownvoteHiddenPostTables.ts (hunk 1 above)
Comment (capJavert) [reply src/migration/1704816100142-RemoveUpvoteDownvoteHiddenPostTables.ts:9]: What if Chrome forgets to parse JavaScript 😆

Comment (capJavert) [reply src/migration/1704816100142-RemoveUpvoteDownvoteHiddenPostTables.ts:9]: image

File: src/schema/posts.ts [hunk 2]
```diff
@@ -1702,59 +1623,6 @@ export const resolvers: IResolvers<any, Context> = {
       }
       return { _: true };
     },
-    upvote: async (
```
Comment (rebelchris) [inline src/schema/posts.ts:1705]: We did double check nowhere used right? (just to be safe)

**Review by rebelchris — APPROVED** [review]
Nothing blocking, just a sanity check 😅

File: src/schema/posts.ts (hunk 2 above)
Comment (capJavert) [reply src/schema/posts.ts:1705]: Yep, frontend and extension use new votePost mutation for months now, also for cancel of upvote/downvote.

Comment (capJavert) [reply src/schema/posts.ts:1705]: Basically only someone on really old extension version.

Comment (rebelchris) [reply src/schema/posts.ts:1705]: That's fine we given them enough time imo.

Comment (omBratteng) [reply src/schema/posts.ts:1705]: Think we're good to go. Neither of these show up in metrics for the last 12 weeks
image

**Review by DragosIorgulescu — APPROVED** [review]
Nothing to add that hasn't been mentioned already. I just wish `Asgard` was spelled `Assgard` so we can labe these branches with `ASS-27`

**Review by idoshamun — APPROVED** [review]
May the force be with you 🙏

**Review by omBratteng — APPROVED** [review]

**Pull request merged**